.github/workflows/release.yml

```diff
@@ -394,8 +394,6 @@ jobs:
         user: __token__
         password: ${{ secrets.PYPI_PYTKET_API_TOKEN }}
         verbose: true
-        # TODO Remove this line when long_description added:
-        verify_metadata: false
```

Comment:
I assume that this is automatically set to true?

Reply:
Yes: https://github.com/pypa/gh-action-pypi-publish#disabling-metadata-verification